components/net/filemanager_thread.rs

```diff
@@ -41,6 +48,8 @@ impl FileManager {
         FileManager {
             receiver: recv,
             idmap: HashMap::new(),
+            classifier: Arc::new(MIMEClassifier::new()),
+            blob_url_store: Arc::new(RwLock::new(BlobURLStore::new())),
```

Comment:
I moved the stuff here (although far from the final model). If this might work, I will change the concrete store previously implemented in script thread to `IpcSender`s.